index.html

```diff
@@ -89,7 +89,90 @@ <h2 class="heading" id="job">Software Developer</h2>
     <br>
     <br>
    <img src="resources/work.png" id="last-img" alt="work">
-    </div>
-
+   <br>
+   <br>
+   <br>
+   <section class="work">
+   <a href="#"><h1 id="heading">My Recent Work</h1></a>
```

Comment:
Please don't use `<h1>` ,instead use `<h2>`, because the recommendation is to have only 1 `<h1>` in the page
Please make the text in UPPERCASE like in Figma